hexrdgui/llnl_import_tool_dialog.py

```diff
@@ -751,8 +789,19 @@ def complete_current_selection(self):
             self.ui.add_transform,
             self.ui.accept_detector,
             enabled=False)
-        self.ui.completed_dets_and_ips.setText(
-            ', '.join(set(self.completed)))
+        self.set_widget_visibility(
+            self.ui.completed_dets_and_ips,
+            visible=True)
+        if self.instrument == 'PXRDIP':
+            shared_file = next(iter(self.loaded_images.values()))
+            self.loaded_images[self.current_image_selection] = shared_file
+        loaded = [
+            f'{d}: {Path(self.loaded_images[d]).stem}'
+            for d in sorted(self.loaded_images)
+        ]
+        text = '\n'.join(set(loaded))
```

Comment:
Is the conversion to a `set` done to remove any duplicates?

Reply:
Yes, but it unnecessary - I've removed it now.